src/pynwb/base.py

```diff
@@ -49,16 +51,32 @@ def add_container(self, **kwargs):
         container.parent = self
 
     @docval({'name': 'container_name', 'type': str, 'doc': 'the name of the NWBContainer to retrieve'})
-    def get_container(self, **kwargs):
+    def get_data_interface(self, **kwargs):
         '''
         Retrieve an NWBContainer from this ProcessingModule
         '''
         container_name = getargs('container_name', kwargs)
         return self.__containers.get(container_name)
 
+    @docval({'name': 'container', 'type': NWBDataInterface, 'doc': 'the NWBDataInterface to add to this Module'})
+    def add_container(self, **kwargs):
```

Comment:
should this function be called ``add_data_interface`` instead for consistency

Reply:
That method exists at line 45

Reply:
Sorry for the oversight. Looks good to me.